ci/cli_merge_sign.sh

```diff
@@ -18,17 +18,26 @@
 
 set -ex
 
+artifacts_dir="${ARTIFACTS_DIR:-?}"
+
 printenv
 
-#TODO(zasgar): Try to make it use another keychain, but it's not a big deal either way
-# since this machine is not used for anythign else.
-security unlock-keychain -p "$JENKINSKEY" login.keychain
+export LANG=C
+export LC_ALL=C
```

Comment:
this seems like it probably isn't needed?

Reply:
Yeah this was from when I was trying to randomly generate the password. Will remove.